resources/views/admin/answers/edit.blade.php

```diff
@@ -29,7 +29,11 @@
                 <select name="genre_id" id="genre" class="form-select">
                     <option value="hidden" disabled>Select Genre</option>
                     @foreach ($all_genres as $genre)
-                        <option value="{{$genre->id}}" key={{$genre->id}}>{{$genre->name}}</option>
+                    @if ($genre->id === $answer->genre_id)
```

Comment:
You can also just use below:

<option {{ $genre->id == $question->genre_id ? 'selected' : '' }} value="{{$genre->id}}">{{$genre->name}}</option>